src/persist-client/src/operators/shard_source.rs

```diff
@@ -119,18 +112,17 @@ where
         as_of,
         until,
         flow_control,
-        consumed_part_rx,
+        completed_fetches_feedback_stream,
         chosen_worker,
         Arc::clone(&key_schema),
         Arc::clone(&val_schema),
     );
-    let (parts, tokens) = shard_source_fetch(
+    let (parts, part_tokens) = shard_source_fetch(
```

Comment:
The naming was already like this before, but I find it mildly confusing that we have:
 - tokens, which are the dataflow token that we return and stash to keep a dataflow alive
 - tokens, which are representations of batches that we have read and can return

We could rename the latter to sth like `completed_fetches_stream`